uiowa/uiowa
# Admissions SEO settings for custom content types

# Updates

* Resolves #3215 
* Resolves #3217 
* Resolves #3218
* Resolves #3216 
* Added `field_meta_tags` to Areas of Study, Scholarship, and Student Profile content types.
* Added description meta tag default overrides for Areas of Study, Scholarship, and Student Profile content types.
* Added relevant new configuration to Admissions and Scholarship splits.
* Re-exported everything.

# How to test

* `blt ds --site=admissions`
* `drush @admissions.local uli admin/people` and masquerade as a webmaster.
* Visit http://admissions.local.drupal.uiowa.edu/node/1141/edit (Area of Study) and confirm you can edit SEO Settings.
* Visit http://admissions.local.drupal.uiowa.edu/node/561/edit (Scholarship) and confirm you can edit SEO Settings.
* Visit http://admissions.local.drupal.uiowa.edu/node/821/edit (Student Profile) and confirm you can edit SEO Settings.
